### PR TITLE
community: support UAE sentence embeddings

### PR DESCRIPTION
**Pull request by SeanLee97**
- **Description:** support loading the current SOTA sentence embeddings [WhereIsAI/UAE](https://huggingface.co/WhereIsAI/UAE-Large-V1) in langchain.
  - **Dependencies:** [angle_emb](https://github.com/SeanLee97/AnglE)
  - **Twitter handle:** @xmlee97

### CODE REVIEW

**Comment by SeanLee97** [comment]
@hwchase17 Could you help review this PR? thanks!

**Comment by hwchase17** [comment]
needs to be merged with master

**Comment by SeanLee97** [comment]
hi @hwchase17 , I've solved the conflicts.

**Review by hwchase17 — APPROVED** [review]
lgtm! thanks

**Comment by SeanLee97** [comment]
When can this branch be merged?

**Comment by hwchase17** [comment]
@SeanLee97 - lint is failing and I do not have permissions to push to this branch

plz either fix lint or give me permissions to push to this branch and I can do that

**Comment by SeanLee97** [comment]
> @SeanLee97 - lint is failing and I do not have permissions to push to this branch
> 
> plz either fix lint or give me permissions to push to this branch and I can do that

hi @hwchase17 , I've granted you the permission. Could you help to push it? many thanks

**Comment by minhduc0711** [comment]
Seems like the relevance scores of retrieved texts are not normalized yet.
```
/data/mdm/miniforge3/envs/rag/lib/python3.11/site-packages/langchain_core/vectorstores.py:311: UserWarning: Relevance scores must be between 0 and 1
```

**Comment by Tejash-Shah** [comment]
Hi, is there any update on this? 
Would like to use `UAE` through `langchain`

File: libs/community/langchain_community/embeddings/__init__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 """
 
 
-import logging
+import logging  # NOQA
```
Comment (baskaryan) [inline libs/community/langchain_community/embeddings/__init__.py:14]: delete comment

**Review by baskaryan — APPROVED** [review]

**Comment by zerodrift** [comment]
Any updates?

**Comment by baskaryan** [comment]
seems to have some lint issues, run 
```bash
cd libs/community
make format
make lint
```

to see if that fixes them

**Comment by efriis** [comment]
wow old one! At this time, we're not recommending custom subclasses of Huggingface implementations for specific models because they're generic. Additionally the integration is managed in https://github.com/langchain-ai/langchain-huggingface instead of here! The `langchain_huggingface.HuggingFaceEmbeddings` version will have the latest and greatest compatibility

**Pull request closed without merging**